src/components/SignIn/SignIn.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import axios from "axios";
+import { Button, TextField, Typography } from "@material-ui/core";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+import { useStyles } from "./SignIn.styles";
+
+export const SignInForm = () => {
+  const classes = useStyles();
+  const formik = useFormik({
+    initialValues: {
+      username: "",
+      password: "",
+    },
+    validationSchema: Yup.object({
+      username: Yup.string()
+        .min(5, "Must be more than 5 characters")
+        .max(15, "Must be less than 15 characters")
+        .required("Required"),
+      password: Yup.string()
+        .min(8, "Must be 8 characters or more")
+        .required("Required"),
+    }),
+    onSubmit: async (values) => {
+      await axios({
+        method: "post",
+        url: "/auth/login",
+        data: JSON.stringify(values, null, 2),
```

Comment:
I think you can send data without parsing it to JSON

Reply:
Done. Replaced with spread values object.